lahja/endpoint.py

```diff
@@ -397,24 +376,29 @@ def subscribed_events(self) -> Set[Type[BaseEvent]]:
         Tell all inbound connections of our new subscriptions
         """
         # make a copy so that the set doesn't change while we iterate over it
-        for inbound_connection in self._inbound_connections.copy():
-            await inbound_connection.notify_subscriptions_updated(self.subscribed_events)
+        subscriptions = self.subscribed_events
+
+        await asyncio.gather(*(
+            remote.notify_subscriptions_updated(subscriptions)
```

Comment:
Noting that this doesn't expose the ability to make this operation block which might be desired.